exasol/nb_connector/itde_manager.py

```diff
@@ -127,47 +146,60 @@ def _get_ipv4_ddresses():
 
 def is_itde_running(conf: Secrets) -> Tuple[bool, bool]:
     """
-    Checks if the ITDE container exists and if it is running. Returns the two boolean
-    flags - (exists, running).
+    Checks if the ITDE container exists and ready to be used. In the Docker Edition that
```

Comment:
Would it be possible to replace the tuple by an enum? 
```python
class ItdeContainerStatus(Enum):
    EXISTS = 0
    RUNNING = 1
```
This would also enable to handle the case `not container_name or not network_name` more explicitly by returning `None` or having an explicit enum value `UNKNOWN` or `INVALID`.

Reply:
I agree, but this would also require changes in the upstream code of the the AI-Lab.

Reply:
I moved the out-of-scope-findings to #93